logger/logger.go

```diff
@@ -6,115 +6,116 @@ import (
 	"io/ioutil"
 	"log"
 	"os"
-	"os/user"
-	"path"
 	"syscall"
 	"time"
 
 	"golang.org/x/net/context"
-	"golang.org/x/oauth2"
-	"golang.org/x/oauth2/google"
-	"google.golang.org/cloud"
+
 	"google.golang.org/cloud/datastore"
 )
 
-// BuildData is used to represent general data for a single invocation of
+// Rusage is the copy from syscall.Rusage for Linux.
+// It is needed because without it the LogRunInfo would fail.
```

Comment:
"would fail" is far from an accurate description! I Would go for:

```go
// It is needed because syscall.Rusage cannot be saved in Datstore because
// it holds unsigned integers. Also, runs are executed inside Docker, so will
// always generate this version of syscall.Rusage.
```